src/utils/get/package.json

```diff
@@ -11,5 +11,9 @@
     "--------------- BUILDING": "----------------------",
     "clean": "rm -rf lib",
     "build": "npm run clean && tsc"
+  },
+  "peerDependencies": {
```

Comment:
this isn't needed in this package

Reply:
Good catch - not sure why that change happened to be honest. Fixed 👍